client/src/components/NewExperimentPage.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchFlags } from "../actions/flagActions";
+import NewExperimentForm from './NewExperimentForm';
+
+const NewExperimentPage = () => {
+  const dispatch = useDispatch();
+  const { flagId } = useParams();
+  const flagData = useSelector((state) =>
+    state.flags.find((flag) => flag.id === +flagId)
+  );
+  const [flagFetched, setFlagFetched] = useState(false);
+  // const metrics = useSelector((metrics) => state.metrics);
+  // const [metricsFetched, setMetricsFetched] = useState(false);
+  const metrics = [
+    { id: 1, name: "Created Account", query_string: "", type: "binomial"},
+    { id: 2, name: "Time on site", query_string: "", type: "count"}
+  ]
+
+  useEffect(() => {
+    if (!flagFetched) {
+      dispatch(fetchFlags());
+      setFlagFetched(true);
+    }
+  }, [dispatch, flagId, flagFetched]);
+
+  // useEffect(() => {
+  //   if (!metricsFetched) {
+  //     dispatch(fetchMetrics());
+  //     setMetricsFetched(true);
+  //   }
+  // }, [dispatch, metrics, metricsFetched]);
+
+  if (!flagData) return null;
+  // if (!flagData || !metrics) return null;
+  return (
+    <div className="ml-5 mt-5">
```

Comment:
Can you add class `w-full` to the top-level div here so that the sidebar doesn't change size when going to the experiment page?